openshift/e2e-tests-openshift.sh

```diff
@@ -86,59 +86,14 @@ function timeout() {
   return 0
 }
 
-function install_knative_serving(){
-  header "Installing Knative serving"
-
-  oc new-project $SERVING_NAMESPACE
-
-  # Deploy Serverless Operator
-  deploy_serverless_operator
-
-  # Install Knative Serving
-  cat <<-EOF | oc apply -f -
-apiVersion: serving.knative.dev/v1alpha1
-kind: KnativeServing
-metadata:
-  name: knative-serving
-  namespace: ${SERVING_NAMESPACE}
-EOF
-
-  # Wait for 4 pods to appear first
-  timeout 900 '[[ $(oc get pods -n $SERVING_NAMESPACE --no-headers | wc -l) -lt 4 ]]' || return
-
-  wait_until_pods_running $SERVING_NAMESPACE || return 1
-
-  wait_until_service_has_external_ip $SERVICEMESH_NAMESPACE istio-ingressgateway || fail_test "Ingress has no external IP"
-
-  wait_until_hostname_resolves "$(kubectl get svc -n $SERVICEMESH_NAMESPACE istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')"
-
-  header "Knative Serving installed successfully"
-}
-
-function deploy_serverless_operator(){
+function install_serverless_operator(){
```

Comment:
I'd just rename the function to install_serverless . You're not only installing the operator but also knative-serving pods etc.

Reply:
updated to `install_serverless`